win32/tests/gballoc_hl_metrics_ut/gballoc_hl_metrics_ut.c

```diff
@@ -0,0 +1,1491 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#ifdef __cplusplus
+#include <cstddef>
+#include <cinttypes>
+#else
+#include <stddef.h>
+#include <inttypes.h>
+#endif
+
+#include "azure_macro_utils/macro_utils.h"
+#include "testrunnerswitcher.h"
+#include "windows.h"
+#include "umock_c/umock_c.h"
+#include "umock_c/umocktypes_stdint.h"
+#include "umock_c/umocktypes_windows.h"
+#include "umock_c/umocktypes.h"
+#include "umock_c/umock_c_negative_tests.h"
+
+#define ENABLE_MOCKS
+#include "azure_c_pal/timer.h"
+#include "azure_c_pal/gballoc_ll.h"
+#undef ENABLE_MOCKS
+
+#include "azure_c_pal/gballoc_hl.h"
+
+#if defined(_MSC_VER)

```

Comment:
already in the win32 folder

Reply:
removed altogether, doesn't actually apply to the file anymore

---
In reply to: [459663761](https://github.com/Azure/azure-c-pal/pull/14#discussion_r459663761) [](ancestors = 459663761)